### ISSUE TITLE
Engine Request - Unity Runtime 

### ISSUE THREAD

**Issue by neuromancer** [dreamer/luxtorpeda#12]
This website has detailed instruction and a list of games: 

https://www.gamingonlinux.com/wiki/Unity_Games_Working_On_Linux_(User_Ported)

Since the process is quite difficult, it will be nice for the users to have everything working using luxtorpeda (for some whitelisted games, of course).

**Comment by d10sfan** [comment]
This one may be tricky because of the licenses involved (unity being closed source).

**Comment by dreamer** [comment]
Not only licenses, but there are also serious technical hurdles and supporting such solution would go seriously out of the scope of Luxtorpeda project. 

If anyone wants to develop a script to address specific game from that list, then the most appropriate place to share them with the community would be writing a game installer for [Lutris](https://lutris.net/). Luxtorpeda packages are (trivially) declarative (do not include arbitrary installation scripts), and I intend to keep them this way. Otherwise, the project would need to require substantially more effort in package maintenance, upgrades would be much harder or even impossible, testing would be harder, and I would need to institute ongoing modding policies instead of doing simple code review.

In my opinion, the proper way of addressing this would be a creation of Unity-reimplementation project, just like [FNA](https://fna-xna.github.io/) did to [XNA](https://en.wikipedia.org/wiki/Microsoft_XNA) - I will keep this request open in case someone finds or develops such project.

I bet most of those titles work fine through Proton. If your game does not and messing with Unity replacement helped you, leave a comment with steam app id, please.

**Comment by neuromancer** [comment]
I agree, Proton runs just fine with Unity games. For the actual open-source replacement, I know [this](https://github.com/osen/mutiny), but sadly it is inactive since 2016.
